pkg/client/vms.go

```diff
@@ -101,9 +101,17 @@ func (service *VMsService) Delete(ctx context.Context, name string) error {
 	return nil
 }
 
-func (service *VMsService) PortForward(ctx context.Context, name string, port uint16) (*websocket.Conn, error) {
+func (service *VMsService) PortForward(
+	ctx context.Context,
+	name string,
+	port uint16,
+	waitSeconds uint16,
```

Comment:
Should the wait time here be optional? Because it is in the API.

Reply:
IMO waiting by default make sense for users who will call `ssh` immediately after VM creation:

```bash
orchard create vm --image ghcr.io/cirruslabs/macos-ventura-base:latest --headless=false ventura-base
orchard ssh vm ventura-base
```

I think it's better UX.